packages/client/src/index.ts

```diff
@@ -54,47 +50,34 @@ export const LogtoSignInSessionItemSchema = type({
 
 export type LogtoSignInSessionItem = Infer<typeof LogtoSignInSessionItemSchema>;
 
-export type HandleRedirect = (url: string) => void;
-
 export default class LogtoClient {
   protected readonly logtoConfig: LogtoConfig;
   protected readonly getOidcConfig = once(this._getOidcConfig);
   protected readonly getJwtVerifyGetKey = once(this._getJwtVerifyGetKey);
 
-  protected readonly storage: Storage;
-  protected readonly requester: Requester;
-  protected readonly handleRedirect: HandleRedirect;
+  protected readonly adapters: ClientAdapters;
```

Comment:
nit: should use singular form `adapter: ClientAdapter`, because we didn't use plural form for `LogtoConfig`, either

Reply:
updated